empty.ts

```diff
@@ -0,0 +1,38 @@
+import { Curry1 } from './utils/types.ts'
+import curryN from './utils/curry_n.ts'
+import { isArray, isString, isObject, isArguments } from './utils/is.ts'
+
+function _empty(x: any) {
+  if(x != null && typeof x.empty === 'function')
+    return x.empty()
+  
+  if(x != null && x.constructor != null && typeof x.constructor.empty === 'function')
+    return  x.constructor.empty()
+  
+  if(isArray(x))
```

Comment:
either put the return statement in same line or use braces

Reply:
Ok